lib/ldap/model/ad/person.rb

```diff
@@ -3,14 +3,14 @@ class AD::Person < Base
     include AD::Timestamps
 
     autoload :Avatar, 'ldap/model/ad/person/avatar'
-
-    validates :sAMAccountName, :givenName, presence: true
+    validates :givenName, presence: true
```

Comment:
`sAMAccountName` *is* required for an `AD::Person`, why you needed to remove it?